example/App.tsx

```diff
@@ -347,7 +366,11 @@ const styles = StyleSheet.create({
       },
     }),
     padding: 16,
-    backgroundColor: COLORS.offWhite,
+    // backgroundColor: COLORS.offWhite,
```

Comment:
I'm not sure if we want to leave it as a comment 😅

Reply:
My mistake, removed it in [46a46ac](https://github.com/software-mansion/react-native-gesture-handler/pull/3055/commits/46a46ac95ce6fa6d33334c362cea374a0e972d6a)